app/models/invenio_rdm_record_converter.rb

```diff
@@ -547,4 +547,32 @@ def format_prism_community_collection_string(mapping_entry)
       community_id
     end
   end
+
+  def original_identifiers(identifiers)
+    identifiers = identifiers.map do |identifier|
+      if identifier.include?("PMID")
+        id = identifier.gsub(/\(PMID\)/, "").strip
+        scheme = "pmid"
+      elsif identifier.include?("DOI")
+        # it's significantly less difficult to just do two removals than find a catch all here
+        id = identifier.gsub(/DOI/, "").gsub(/[():]/, "").strip
```

Comment:
:crossed_fingers: there is no DOI called "10.1234/IDOIT_CAUSE_I_CAN" ;)

Reply:
That would be miserable lol. I need to take a class on common techniques for dealing with string parsing and identification. I'm sure these kinds of things come up for other programmers too.